__DEFINES/setup.dm

```diff
@@ -782,6 +782,7 @@ SEE_PIXELS	256
 #define MAT_BRASS   	"$brass"
 #define MAT_RALLOY   	"$ralloy"
 #define MAT_ICE			"$ice"
+#define MAT_MYTHRIL		"&mythrl"
```

Comment:
the other macros are using `$` instead of `&`

Reply:
shouldn't this be `$mythril` for that matter